User Interface/Tooltips/source/State_DemoCode.hx

```diff
@@ -0,0 +1,133 @@
+import flixel.addons.ui.Anchor;
+import flixel.addons.ui.BorderDef;
+import flixel.addons.ui.FlxUIButton;
+import flixel.addons.ui.FlxUISprite;
+import flixel.addons.ui.FlxUIText;
+import flixel.addons.ui.FlxUITooltip;
+import flixel.addons.ui.FlxUITooltip.FlxUITooltipStyle;
+import flixel.addons.ui.FlxUITypedButton;
+import flixel.addons.ui.FontDef;
+import flixel.FlxSprite;
+import flixel.FlxState;
+import flixel.math.FlxPoint;
+import flixel.text.FlxText;
+import flixel.util.FlxColor;
+import haxe.xml.Fast;
+import flash.Lib;
+import flixel.FlxG;
+import flixel.addons.ui.FlxUIState;
+import openfl.Assets;
+import openfl.text.TextFormat;
+import flixel.text.FlxText.FlxTextBorderStyle;
+import openfl.text.TextFormatAlign;
+/**
+ * @author Lars Doucet
+ */
+
+class State_DemoCode extends FlxUIState
+{
+
+	override public function create() 
+	{
+		_xml_id = "state_code";
+		super.create();
+		makeByHand();
+	}
+	
+	override public function getEvent(id:String, sender:Dynamic, data:Dynamic, ?params:Array<Dynamic>):Void 
+	{
+		super.getEvent(id, sender, data, params);
+		switch(id)
+		{
+			case FlxUITypedButton.CLICK_EVENT: 
+				var str:String = (params != null && params.length >= 1) ? cast params[0] : "";
+				if (str == "defaults")
+				{
+					FlxG.switchState(new State_Demo2());
+				}
+				if (str == "no_defaults")
+				{
+					FlxG.switchState(new State_Demo());
+				}
+		}
+	}
+	
+	private function makeByHand():Void
+	{
+		_ui.addAsset(cast new FlxUISprite().makeGraphic(FlxG.width, FlxG.height, 0xFF404040), "bkg");
+		
+		var border  = new BorderDef(FlxTextBorderStyle.OUTLINE, FlxColor.BLACK, 1, 1);
+		var sans8   = new FontDef("vera", ".ttf", "assets/fonts/verab.ttf", new TextFormat(null, 8, FlxColor.WHITE, true));
```

Comment:
Seeing how the font size is almost the only thing varying here, you could have a `createFontDef()` helper function to get rid of the redundancy.